ui/config-overrides/index.js

```diff
@@ -0,0 +1,8 @@
+/* global module */
+
+module.exports = function override(config, _env) {
```

Comment:
ESM doesn't work?

Reply:
Not with `react-app-rewired` unfortunately